packages/cli/src/extract.ts

```diff
@@ -90,6 +90,28 @@ function processFile(
 ) {
   let messages: ExtractedMessageDescriptor[] = [];
   let meta: Record<string, string> | undefined;
+  const {onMetaExtracted, onMsgExtracted} = opts;
+
+  opts = {
+    ...opts,
+    onMsgExtracted(_, msgs) {
+      if (opts.extractSourceLocation) {
+        msgs = msgs.map(msg => ({
+          ...msg,
+          ...calculateLineColFromOffset(source, msg.start),
+        }));
+      }
+      messages = messages.concat(msgs);
+
+      if (onMsgExtracted) onMsgExtracted(_, msgs);
```

Comment:
hmm do u need this? `extract` gives u all the extracted messages at the end

Reply:
Without this code it does not. With the current code, if I don't pass in a `idInterpolationPattern` value it returns `"{}"`

Reply:
Unless you mean just the callback? I just put it there because I wasn't sure if/where someone might want to pass in their own `onMsgExtracted`. If you think it's not necessary I can remove this line.

Reply:
yeah if u can rm that & `onMetaExtracted` I think that's gtg.

Reply:
Done and done